build/common.props

```diff
@@ -5,7 +5,7 @@
     <LangVersion>latest</LangVersion>
     <MajorVersion>4</MajorVersion>
     <MinorVersion>1035</MinorVersion>
-    <PatchVersion>1</PatchVersion>
+    <PatchVersion>4</PatchVersion>
```

Comment:
previously patch version updates were made directly to the release branch. But I observed release team is following the practice to update them in the working branch. So doing the same.